versions.lock

```diff
@@ -129,32 +129,32 @@ org.apache.kerby:kerby-util:1.0.1 (2 constraints: 6518bdb6)
 org.apache.logging.log4j:log4j-api:2.13.2 (4 constraints: d033f2b0)
 org.apache.logging.log4j:log4j-core:2.13.2 (2 constraints: 0d16b224)
 org.apache.logging.log4j:log4j-slf4j-impl:2.13.2 (1 constraints: 3a053a3b)
-org.apache.lucene:lucene-analysis-common:9.0.0-SNAPSHOT (10 constraints: ceb88a15)
```

Comment:
Changes in this file are done by `gradlew --write-locks`, it's not committer's responsibility!